### PR TITLE
Convert vector debug pages to ESM

### PR DESCRIPTION
**Pull request by jonkoops**
Converts the vector debug pages (located in `debug/vector`) to ESM and cleans up the code in the process. This is part of a greater effort to land ESM as a first-class citizen. Code that was broken has been replaced with working code or removed.

### CODE REVIEW

File: debug/vector/blanket.html [hunk 1]
```diff
@@ -1,70 +1,62 @@
...
+			});
+
+			new DebugBlanket({
+				padding: -0.1,
+				continuous: true
+			}).addTo(map);
+
+			const canvas = new Canvas({
+				padding:-0.1,
+				// continuous: true
```
Comment (Falke-Design) [inline debug/vector/blanket.html:49]: `continuous` is not a valid option and should be removed https://leafletjs.com/reference.html#canvas

File: debug/vector/blanket.html [hunk 2]
```diff
@@ -1,70 +1,62 @@
...
+			}).addTo(map);
+			canvas._container.style.border = '2px solid red';
+			new CircleMarker([40.5, -3.6], {radius: 20, color: 'red', renderer: canvas}).addTo(map);
+
+			const svg = new SVG({
+				padding:-0.1,
+				// continuous: true,
```
Comment (Falke-Design) [inline debug/vector/blanket.html:56]: `continuous` is not a valid option and should be removed https://leafletjs.com/reference.html#canvas

**Review by Falke-Design — APPROVED** [review]
Besides the two points, which has nothing to do with this changes, everything looks good

**Comment by jonkoops** [comment]
Went ahead and fixed the review comments as well, I'll merge this in :+1: Thanks again @Falke-Design!

**Pull request merged**